src/plugins/booth.js

```diff
@@ -180,7 +180,7 @@ class Booth {
     return this.uw.redis.lrange('waitlist', 0, -1);
   }
 
-  async publish(next) {
+  async publish(next, previous) {
```

Comment:
This is most likely be done adpter side for the bot so is not necessary, you can remove the changes in this file